### PR TITLE
Noto Sans Mongolian: Version 3.000; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by simoncozens**
ed1b187: [gftools-packager] Noto Sans Mongolian: Version 3.000; ttfautohint (v1.8.4.7-5d5b) added

* Noto Sans Mongolian Version 3.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/mongolian.git at commit https://github.com/notofonts/mongolian/commit/.

### CODE REVIEW

**Comment by simoncozens** [comment]
```
This release is a completely re-engineered font, bringing the shaping behaviour into compliance with the draft UTN for Mongolian shaping by Kushim Jiang.
```

> FAIL Please remove 'latin-ext' from METADATA.pb since none of its glyphs are supported by this font file. 

I didn't put it there, `gftools-add-font` did. Silly add-font. I'll take it out manually.

> 🔥 FAIL: Whitespace glyphs have ink? ([com.google.fonts/check/whitespace_ink](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/whitespace_ink))
> 🔥 FAIL Glyph 'mvs' has ink. It needs to be replaced by an empty glyph. [code: has-ink]
> 🔥 FAIL Glyph 'nnbsp' has ink. It needs to be replaced by an empty glyph. [code: has-ink]

These glyphs are used for Mongolian shaping, and will be substituted. Please leave as they are.

> ⚠ WARN: Check for codepoints not covered by METADATA subsets. 

There are some real problems here. We will need to add MVS4 to the Mongolian definition, and I'll manually add chinese-simplified to the METADATA.pb.

**Comment by simoncozens** [comment]
Updated

## Noto Sans Mongolian: Version 3.001; ttfautohint (v1.8.4.7-5d5b) added

---
 004ff44: [gftools-packager] Noto Sans Mongolian: Version 3.001; ttfautohint (v1.8.4.7-5d5b) added

* Noto Sans Mongolian Version 3.001; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/mongolian.git at commit https://github.com/notofonts/mongolian/commit/.

**Comment by simoncozens** [comment]
Latin Kernel -> Latin Core. Should be good now.

**Comment by emmamarichal** [comment]
Same fontbakery version between the two last reports, but this fail appeared:
`🔥 FAIL: Check that legacy accents aren't used in composite glyphs. (derived from [com.google.fonts/check/legacy_accents](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/legacy_accents)) (com.google.fonts/check/legacy_accents)`

Is it related to this?

> Latin Kernel -> Latin Core.

___________
Here is the before/after. I can't really know about mongolian script changes, so I guess it's part of the update.
What about the circled numbers?

diffbrowsers_glyphs-Windows-10-10 0 20348-SP0_firefox_119 0


https://github.com/google/fonts/assets/64773544/81211e1a-13d6-4c58-a243-e85f64fa16fb



___________
These kind of details have been updated in the other Notos updates. Maybe related to this: _Check that legacy accents aren't used in composite glyphs._   Should we add some anchors here?

**New version:**

Capture d’écran 2023-11-15 à 10 16 07

**Comment by simoncozens** [comment]
>  Check that legacy accents aren't used in composite glyphs.

Grr. This check is not relevant, and so is confusing. It is only relevant to *precomposed* mark glyphs. It checks that a precomposed letter like e.g. `eacute` is made up of components `e` and `eacutecomb`, not `e` and `acute`. But if those are literally the same outlines, then it doesn't matter.

I'll check why Vietnamese is not working, but as this is only a Latin Core glyphset I don't think Vietnamese support is in scope anyway.

File: ofl/notosansmongolian/METADATA.pb [hunk 1]
```diff
@@ -10,15 +10,17 @@ fonts {
...
   full_name: "Noto Sans Mongolian Regular"
-  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/mongolian)"
+  copyright: "Copyright 2023 The Noto Project Authors (https://github.com/notofonts/mongolian)"
 }
 subsets: "latin"
 subsets: "latin-ext"
+subsets: "math"
 subsets: "menu"
 subsets: "mongolian"
+subsets: "symbols"
```
Comment (RosaWagner) [inline ofl/notosansmongolian/METADATA.pb:20]: @simoncozens did you say something about adding the chinese simplified subsets?

**Comment by RosaWagner** [comment]
Vietnamese is not expected to work here. @simoncozens Do we need to wait for the subset to include the missing mongolian codepoint in the glyphsets repo?

**Comment by simoncozens** [comment]
I've added chinese-simplified. We do need Mongolian FVS4 added to the subset - eng team to merge https://github.com/googlefonts/glyphsets/pull/143 and update internally.

**Review by emmamarichal — APPROVED** [review]

**Pull request merged**